client/.babelrc

```diff
@@ -3,7 +3,7 @@
     [
       "@babel/preset-env", {
         "targets": {
-          "electron": "26"
+          "electron": "34"
```

Comment:
[future improvement] We could check this automatically from package.json, couldn't we?

Reply:
Not sure, did not investigate.